src/lib/github/addIssuesToRepo.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ *  API Function that adds issues with comments to a repository
+ */
+
+export interface Issue {
+    title: string;
+    body: string[];
+ }
+
+ export default async function addIssuesToRepo(context: any, issues: Issue[] ): Promise<any> {
+    const octokit = context.github;
+    const result: any = issues.map( async (issue: Issue) => {
```

Comment:
The type of any here is wrong.It should be a `Promise` related type.

Reply:
@lubegasimon its not exactly wrong per say, I just think it would be better to use a more precise type.